doc/content/getting-started/migrating/migrating-from-ce-to-ch/establish-new-session/_index.md

```diff
@@ -0,0 +1,114 @@
+
+## ABP
+
+Migrating an ABP device without persisting its active session means its session and MAC state parameters will be reset, or the device will be completely re-programmed with new session parameters.
+
+{{< note >}} Migrating devices without persisting active sessions is the preferred migration method. For ABP devices, we also advise to re-program the device using session parameters generated on {{% tts %}} Cloud, if possible. For devices that can be re-programmed, it is also recommended to change their activation method from ABP to OTAA. {{</ note >}}
+
+Let's first assume that you are able to re-program your ABP device with new DevAddr, NwkSKey and AppSKey issued by {{% tts %}} Cloud. Since {{% tts %}} Community Edition and {{% tts %}} Cloud are both connected to Packet Broker, and your ABP device will be re-programmed with a new DevAddr from {{% tts %}} Cloud DevAddr block, Packet Broker will be able to route your device's traffic to {{% tts %}} Cloud even if your gateway stays connected to {{% tts %}} Community Edition, i.e. you don't have to migrate your gateway to {{% tts %}} Cloud, only your device. However, if you want to migrate your gateway to {{% tts %}} Cloud too, see [Migrating Gateways]({{< ref "/getting-started/migrating/gateway-migration" >}}) for instructions.
+
+The process of migrating your ABP device from {{% tts %}} Community Edition to {{% tts %}} Cloud without its active session and by re-programming it is the most straightforward if you use the {{% tts %}} Console.
+
+Create a new ABP device on {{% tts %}} Cloud. See [Adding Devices]({{< ref "/devices/adding-devices" >}}) for instructions on creating a device. Generate new DevAddr, AppSKey and NwkSKey values while creating the device.
+
+When the device is created, navigate to **General settings &#8594; Network Layer &#8594; Advanced MAC settings** and enable the **Resets frame counters** option.
+
+Now reset your ABP device. You should immediately see uplinks arriving from your device.
+
+However, if you don't want to re-program your device, i.e. you want to keep the DevAddr, AppSKey and NwkSKey from {{% tts %}} Community Edition, read the instructions below.
+
+---
+
+Now let's assume that you want to migrate your ABP device to {{% tts %}} Cloud in such manner that it keeps its DevAddr, NwkSKey and AppSKey it was programmed with upon its registration on {{% tts %}} Community Edition network. Note that {{% tts %}} Community Edition and {{% tts %}} Cloud use different DevAddr blocks. Since Packet Broker routes traffic according to the DevAddr blocks, in this case it won't be able to route your device's traffic properly. To successfully migrate your ABP device in this case, you also need to migrate your gateway to {{% tts %}} Cloud. See instructions for [Migrating Gateways]({{< ref "/getting-started/migrating/gateway-migration" >}}). The ideal scenario would be to migrate your gateway and your device simultaneously.
```

Comment:
Doesn't this part fall under "migrating with active session"?

Reply:
I had doubts about that, because Aggelos explained to me months ago that the "session" in this context implies more than just DevAddr, NwkSKey and AppSKey. Since this process needs a frame counters reset, I thought that the idea of a "session" is broken by that. I can move it if that's not really true.